### PR TITLE
Added few more examples for Text Modifiers & layout priority.

### PR DESCRIPTION
**Pull request by meetAhmed**
Hi **Barbara**, thank you for this awesome repo.

> This pull request includes the following additions  and updates:

Examples Added:
- ActionSheet
- Confirmation dialog
- ZIndex
- Layout priority
- Offset
- Fixed size view

Updates:
- ContentView to reflect the new examples.
- Added a few text modifiers for enhanced text styling.

### CODE REVIEW

**Comment by barbaramartina** [comment]
Hi @meetAhmed, I was on holidays for some weeks, I will review your contribution. Thanks for adding it.

**Review by barbaramartina — CHANGES_REQUESTED** [review]
thanks for your contributions, I have reviewed it and most of the examples are already in (ActionSheet, ZIndex, Layout priority, Offset, Fixed size view). For modifiers we do not create new views, there is a section about how to contribute in the readme, I moved it up in the text, so it is more visible.

The examples for text selection are great, could you adjust the PR to leave this contribution please? I would like to merge that part.

Confirmation Dialog, is a custom view, not a standard component so I prefer to not include it unless it is made reusable and customizable, as the other views in the reusable views section.

File: SwiftUICatalog/Views/Sections/View Layouts/OffsetView.swift [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import SwiftUI
```
[the remark this answers is not in the document]
Comment (meetAhmed) [reply SwiftUICatalog/Views/Sections/View Layouts/OffsetView.swift]: OffsetView file removed.

File: SwiftUICatalog/Views/Sections/View Layouts/ZIndexView.swift [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import SwiftUI
+
+struct ZIndexView: View, Comparable {
```
[the remark this answers is not in the document]
Comment (meetAhmed) [reply SwiftUICatalog/Views/Sections/View Layouts/ZIndexView.swift]: ZIndexView file removed.

File: SwiftUICatalog/Views/Sections/View Layouts/FixedSizeExampleView.swift [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import SwiftUI
+
+struct FixedSizeExampleView: View, Comparable {
```
[the remark this answers is not in the document]
Comment (meetAhmed) [reply SwiftUICatalog/Views/Sections/View Layouts/FixedSizeExampleView.swift]: FixedSizeExampleView file removed.

File: SwiftUICatalog/Views/Sections/View Layouts/ActionSheetView.swift [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                    name: "Ahmed Ali",
+                    link: "https://github.com/meetAhmed"
+                )
+                .padding(.top, 80)
+            }
+        })
+    }
+}
+
+private extension ActionSheetView {
```
[the remark this answers is not in the document]
Comment (meetAhmed) [reply SwiftUICatalog/Views/Sections/View Layouts/ActionSheetView.swift]: ActionSheetView file removed.

**Comment by meetAhmed** [comment]
Hi @barbaramartina, I have removed duplicated code. Please take a look when you get some time. Thank you.

**Review by barbaramartina — APPROVED** [review]

**Comment by barbaramartina** [comment]
Thanks for your contribution, I am incorporating it today to the main code. Have a good weekend.

**Pull request merged**